flink-shaded-zookeeper-parent/flink-shaded-zookeeper-34/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,13 @@
+flink-shaded-zookeeper-3
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.google.guava:guava:27.0.1-jre
+- org.apache.curator:curator-client:4.2.0
```

Comment:
where comes from these curator 4.x dependencies?

Reply:
`curator-recipes` pulls it in.

Reply:
Aha I miss that it is in `flink-shaded` repo and the curator version is 4.x.